src/components/Navbar.js

```diff
@@ -50,10 +53,10 @@ const ButtonRegister = ({ isLoggedIn }) => {
 
 /** Displays either notification button or empty div
  * @param {object} props
- * @param {boolean} props.isLoggedIn
+ * @param {boolean} props.isAuthenticated
  */
-const ButtonNotification = ({ isLoggedIn }) => {
-  if (isLoggedIn) {
+const ButtonNotification = ({ isAuthenticated }) => {
+  if (isAuthenticated) {
```

Comment:
You don't even need to pass that as a prop.
const { isAuthenticated } = useUserAuthenticationData();

Reply:
Fixed